File: Software_University/04_PHP_Web_Development_Basics/10) OOP Basics - Exercises/06. Raw Data/Engine.php
<?php

namespace Van\Classes;

class Engine
{
    /**
     * @var int
     */
    private $speed;

    /**
     * @var int
     */
    private $power;

    /**
     * Engine constructor.
     *
     * @param int $speed
     * @param int $power
     */
    public function __construct(int $speed, int $power)
    {
        $this->speed = $speed;
        $this->power = $power;
    }

    /**
     * @return int
     */
    public function getSpeed(): int
    {
        return $this->speed;
    }

    /**
     * @param int $speed
     */
    public function setSpeed(int $speed)
    {
        $this->speed = $speed;
    }

    /**
     * @return int
     */
    public function getPower(): int
    {
        return $this->power;
    }

    /**
     * @param int $power
     */
    public function setPower(int $power)
    {
        $this->power = $power;
    }
}

File: Software_University/09_Java_Advanced_January_2019/14. Defining Classes - Exercises/04. Raw Data/CarParts/Engine.java
package CarParts;

public class Engine {

    private Integer speed;

    private Integer power;

    public Engine(Integer speed, Integer power) {
        setSpeed(speed);
        setPower(power);
    }

    public Integer getSpeed() {
        return speed;
    }

    private void setSpeed(Integer speed) {
        this.speed = speed;
    }

    public Integer getPower() {
        return power;
    }

    private void setPower(Integer power) {
        this.power = power;
    }
}
